### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by HokageHEL**
[DEMO LINK](https://HokageHEL.github.io/react_add-todo-form/)
I messed up a bit and made a component for TodoForm.  I hope this won't be a big issue

### CODE REVIEW

**Review by etojeDenys — CHANGES_REQUESTED** [review]
i can create the todo only with spaces, let's fix that
image

File: src/components/Services/User.ts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import { User } from '../../Types/User';
+import usersFromServer from '../../api/users';
+
+export function getUserById(userId: number): User | null {
+  return usersFromServer.find(user => user.id === userId)
+    || null;
+}
```
Comment (Esceype) [inline src/components/Services/User.ts:7]: Why do you create Services inside components folder, is it a component?

File: src/components/TodoInfo/TodoInfo.tsx [hunk 2]
```diff
@@ -1 +1,20 @@
...
+
+type Props = {
+  todo: Todo;
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => (
+  <article
+    className={`
+      TodoInfo
+      ${todo.completed && 'TodoInfo--completed'}`}
```
Comment (Esceype) [inline src/components/TodoInfo/TodoInfo.tsx:13]: Use classNames lib please

File: src/components/UserInfo/UserInfo.tsx [hunk 3]
```diff
@@ -1 +1,12 @@
...
+
+type Props = {
+  user: User;
+};
+
+export const UserInfo: React.FC<Props> = ({ user }) => (
+  <a className="UserInfo" href={`mailto:${user.email}`}>
+    {user.name}
+  </a>
+);
```
Comment (Esceype) [inline src/components/UserInfo/UserInfo.tsx:12]:
```suggestion
export const UserInfo: React.FC<Props> = ({ user }) => {
  const { name, email } = user;

  return (
    <a className="UserInfo" href={`mailto:${email}`}>
      {name}
    </a>
  );
};```

File: src/components/TodoInfo/TodoInfo.tsx [hunk 4]
```diff
@@ -1 +1,20 @@
...
+    data-id={todo.id}
+  >
+    <h2 className="TodoInfo__title">{todo.title}</h2>
+
+    {todo.user && <UserInfo user={todo.user} />}
```
Comment (Esceype) [inline src/components/TodoInfo/TodoInfo.tsx:18]: Destructure the todo in this component, since you're using more than one of its keys

**Review by Esceype — CHANGES_REQUESTED** [review]
At all great job!!!
Left some minor comments.

**Review by etojeDenys — APPROVED** [review]
good job

**Pull request closed without merging**